backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -171,7 +163,26 @@ public static List<FeedbackMessage> validateTestResult(ValueOrError input) {
   }
 
   public static List<FeedbackMessage> validateSpecimenType(ValueOrError input) {
-    return validateSpecificValueOrSNOMED(input, SPECIMEN_TYPE_VALUES);
+    List<FeedbackMessage> errors = new ArrayList<>();
+    String value = parseString(input.getValue());
+
+    if (value == null) {
+      return errors;
+    }
+
+    boolean nonSNOMEDValue = value.matches(ALPHABET_REGEX);
+
+    if (nonSNOMEDValue) {
+      return errors;
+    }
```

Comment:
Neither of these checks are actually necessary are they?  With the required fields validation and the value replacement already having taken place?